go.mod

```diff
@@ -57,7 +59,6 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-	k8s.io/apiextensions-apiserver v0.23.0 // indirect
```

Comment:
suggest: I guess go.mod needs to be formatted as a separate commit.

Reply:
no, I need to add apiextensions module to be able to handle CRD's, which are defined (schema and types) in that module